docs/minimum-versions.md

```diff
@@ -50,10 +50,14 @@ https://help.keyman.com/developer/engine/android/latest-version/
 |-----------------------------------|--------------|
 | KEYMAN_DEFAULT_VERSION_UBUNTU_CONTAINER | noble |
 | KEYMAN_MIN_TARGET_VERSION_ANDROID | 5 |
```

Comment:
```suggestion
| KEYMAN_MIN_TARGET_VERSION_ANDROID | 5.0 |
```

I should have made this clearer so not to confuse 5.1 (SDK 22)

Reply:
nvm - will suggestion on the .sh

Reply:
minimum versions are correct for mac and ios but I expect them to change before we release 18 as we make the move to Xcode 16